lib/_tls_wrap.js

```diff
@@ -847,9 +847,25 @@ Server.prototype.setOptions = function(options) {
   }
 
   if (options.pfx) this.pfx = options.pfx;
-  if (options.key) this.key = options.key;
+  var defaultCiphers = options.ciphers == tls.DEFAULT_CIPHERS;
+  if (!options.key) {
+    if (options.ciphers == undefined || defaultCiphers) {
```

Comment:
It's kind of an unspoken rule and won't matter in this case, but what @thefourtheye means is that generally it's better to be strict and use `===` in JS to avoid language deficits :wink:.